backend/plugin/db/cosmosdb/cosmosdb.go

```diff
@@ -131,201 +127,19 @@ func (driver *Driver) QueryConn(ctx context.Context, _ *sql.Conn, statement stri
-					values[columnIndexMap[k]] = &v1pb.RowValue{Kind: &v1pb.RowValue_DoubleValue{DoubleValue: v}}
-				}
-			case bool:
-				values[columnIndexMap[k]] = &v1pb.RowValue{Kind: &v1pb.RowValue_BoolValue{BoolValue: v}}
-			case map[string]any:
-				// Handle nested objects if necessary
-				// Convert to JSON string representation for example
-				jsonBytes, err := json.Marshal(v)
-				if err != nil {
-					slog.Warn("failed to marshal JSON", slog.Any("object", v), log.BBError(err))
-					illegal = true
-					break
-				}
-				values[columnIndexMap[k]] = &v1pb.RowValue{Kind: &v1pb.RowValue_StringValue{StringValue: string(jsonBytes)}}
-			case []any:
-				// Handle arrays if necessary
-				// Convert to JSON string representation for example
-				jsonBytes, err := json.Marshal(v)
-				if err != nil {
-					slog.Warn("failed to marshal JSON", slog.Any("array", v), log.BBError(err))
-					illegal = true
-					break
-				}
-				values[columnIndexMap[k]] = &v1pb.RowValue{Kind: &v1pb.RowValue_StringValue{StringValue: string(jsonBytes)}}
-			case nil:
-				values[columnIndexMap[k]] = &v1pb.RowValue{Kind: &v1pb.RowValue_NullValue{}}
-			default:
-				// Handle unknown types
-				values[columnIndexMap[k]] = &v1pb.RowValue{Kind: &v1pb.RowValue_StringValue{StringValue: "unknow"}}
-			}
-			for i := 0; i < len(values); i++ {
-				if values[i] == nil {
-					values[i] = &v1pb.RowValue{Kind: &v1pb.RowValue_NullValue{}}
-				}
-			}
-		}
-		if illegal {
-			break
-		}
+	for _, item := range items {
 		result.Rows = append(result.Rows, &v1pb.QueryRow{
-			Values: values,
+			Values: []*v1pb.RowValue{
+				{Kind: &v1pb.RowValue_StringValue{StringValue: string(item)}},
+			},
 		})
 	}
-	result.Latency = durationpb.New(time.Since(startTime))
 
-	if illegal {
-		var rows []*v1pb.QueryRow
-		for _, item := range items {
-			rows = append(rows, &v1pb.QueryRow{
-				Values: []*v1pb.RowValue{
-					{Kind: &v1pb.RowValue_StringValue{StringValue: string(item)}},
-				},
-			})
-		}
-		return []*v1pb.QueryResult{
-			{
-				Rows: rows,
-			},
-		}, nil
-	}
+	result.Latency = durationpb.New(time.Since(startTime))
 
 	return []*v1pb.QueryResult{result}, nil
 }
-
-func getColumns(unmarshalledItems []map[string]any) (columnNames []string, columnTypes map[string]string, columnIndexMap map[string]int) {
-	columnNamesSet := make(map[string]bool)
-	columnTypes = make(map[string]string)
-
-	for _, m := range unmarshalledItems {
-		for k, v := range m {
-			if _, ok := columnNamesSet[k]; ok {
-				continue
-			}
-			columnNamesSet[k] = true
-			columnTypes[k] = getType(v)
-		}
-	}
-	columnNames, columnIndexMap = getOrderedColumns(columnNamesSet)
-	return columnNames, columnTypes, columnIndexMap
-}
-
-func getOrderedColumns(columnSet map[string]bool) ([]string, map[string]int) {
-	var columns []string
-	for k := range columnSet {
-		columns = append(columns, k)
-	}
-	// Put built-in columns at the end.
-	builtInColumns := map[string]bool{
```

Comment:
For better UX, frontend can take this responsibility to put built-in column at the end like cosmos studio does.

Reply:
Yes we do the sorting in the frontend in this PR